config/locales/en.yml

```diff
@@ -106,6 +106,10 @@ en:
         other: "Warnings"
       work: "Work"
     attributes:
+      archive_warning:
```

Comment:
Should I also update lines 104-106? I have no idea if it's used anywhere, but the model is no longer named Warning, so presumably something somewhere might be broken...

Reply:
I'd say go for it. There's no warning model, so at best it will help, and at worst, it can't hurt.